config/sync/block_content.type.employee_pop_up.yml

```diff
@@ -0,0 +1,8 @@
+uuid: c9416ddd-896b-4735-b0f3-abbb0489a368
+langcode: en
```

Comment:
Add config/sync to gitignore.  All the files under config/sync should be part of gitignore and  should be part of PR.

Reply:
@anweshasaha9 I can still see config/sync file in PR. Product content type & Product Features paragraph configurations are missing in the missing.